src/components/cursor.js

```diff
@@ -275,7 +275,11 @@ module.exports.Component = registerComponent('cursor', {
     }
 
     // Unset current intersection.
-    if (this.intersectedEl) { this.clearCurrentIntersection(); }
+    if (this.intersectedEl) {
+      this.clearCurrentIntersection();
+      // Already intersecting.
+      if (this.intersectedEl === intersectedEl) { return; }
```

Comment:
It isn't enough, because 'clearCurrentIntersection' updates 'this.intersectedEl' to another raycast element.

Reply:
This exact line is at L272, before the `clearCurrentIntersection` is ever called, right?

Reply:
At L272, of course, `this.intersectedEl` and `intersectedEl` aren't same because `this.intersectedEl` is previous intersected element.
`this.intersectedEl` may be changed to what same as `intersectedEl` by `setIntersection` called in `clearCurrentIntersection`.
Since this, it is necessary to check equality again after calling `clearCurrentIntersection` to avoid calling 'setIntersection' twice.

Reply:
Haha okay I see. Thanks for explaining

Reply:
What about moving this check inside the `setIntersection` method? If the intersected method is the same as current do nothing